codalab/client/local_bundle_client.py

```diff
@@ -928,7 +943,8 @@ def update_worksheet_items(self, worksheet_info, new_items):
         last_item_id = worksheet_info['last_item_id']
         length = len(worksheet_info['items'])
         worksheet = self.model.get_worksheet(worksheet_uuid, fetch_items=False)
-        check_worksheet_has_all_permission(self.model, self._current_user(), worksheet)
+        if not spec_util.is_dashboard(worksheet_info['name']):
```

Comment:
Yeah this is definitely not ideal. I put in these checks because the first user who visit the dashboard needs to create it as a root user, but he doesn't have the permission to populate the dashboard after creating it. So shall we make the creation of the dashboard part of the configuration process, not specific to any user? I think that might make more sense. If so, could you point me to some files that I should look at to achieve that?